platforms/hyperledger-indy/configuration/roles/setup/vault_kubernetes/tasks/main.yaml

```diff
@@ -58,7 +58,7 @@
 - name: Write reviewer token for Organizations
   shell: |
     export REVIEWER_TOKEN=$(KUBECONFIG={{ kubernetes.config_file }} kubectl get secret $(KUBECONFIG={{ kubernetes.config_file }} kubectl get serviceaccount {{ organization }}-admin-vault-auth -n {{ component_ns }} -o jsonpath={.secrets[0].name}) -n {{ component_ns }} -o  jsonpath={.data.token} | base64 --decode)
-    vault write auth/{{ auth_path }}/config token_reviewer_jwt="$REVIEWER_TOKEN" kubernetes_host=$(KUBECONFIG={{ kubernetes.config_file }} kubectl config view -o jsonpath="{.clusters[?(@.name==\"{{ kubernetes.context }}\")].cluster.server}") kubernetes_ca_cert=@"./build/{{ organization }}.ca.cert"
+    vault write auth/{{ auth_path }}/config token_reviewer_jwt="$REVIEWER_TOKEN" kubernetes_host="$(KUBECONFIG={{ kubernetes.config_file }} kubectl config view --minify | grep server | cut -f 2- -d ":" | tr -d " ")" kubernetes_ca_cert=@"./build/{{ organization }}.ca.cert"
```

Comment:
Will this work on any server (is minify need to be installed)

Reply:
Removed line (red) worked on our old AWS instance. For now, we have a new instance and it stopped working. The new line (green) works on AWS and also on Minikube on Mac OSX. I installed nothing more on my Mac OSX except what is added in documentation (JQ, GNU Tar and Kubernetes for Ansible)